docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -516,9 +591,155 @@ The ``feed.list`` command displays available feed functions in the Cortex.
 
 **Syntax:**
 
-
 .. storm-cli:: feed.list --help
 
+
+.. _storm-gen:
+
+gen
+---
+
+Storm includes various ``gen.*`` ("generator") commands that allow you to easily query
```

Comment:
Might just go with "generate" here, a "generator" is a type of function in Python/Storm which may be confusing.

Reply:
Switched to "generate" :+1: